apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -75,6 +75,23 @@ function add_public_coming_soon_to_settings_endpoint_post( $input, $unfiltered_i
 }
 add_filter( 'rest_api_update_site_settings', __NAMESPACE__ . '\add_public_coming_soon_to_settings_endpoint_post', 10, 2 );
 
+/**
+ * This filter adds the coming soon page to the list of page templates using the `vertical_templates` hook
+ *
+ * @param array $templates The array of all available page templates.
+ *
+ * @return array The updated $templates array
+ */
+function add_public_coming_soon_page_template( $templates ) : array {
+	$templates[] = json_decode(
+		// phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
+		file_get_contents( __DIR__ . '/coming-soon-page-template.json' ),
+		false
+	);
+	return $templates;
+}
+add_filter( 'vertical_templates', __NAMESPACE__ . '\add_public_coming_soon_page_template' );
```

Comment:
Would it make more sense to add a [new filter here](https://github.com/Automattic/wp-calypso/blob/587bef31a70e978d1558739d4a741bdec1e196b5/apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/class-starter-page-templates.php#L223)? 

That would have translation implications but I don't think this will be translated via this hook anyway.

And also heads up we could use patterns for this soon too #45926